config/initializers/sentry.rb

```diff
@@ -6,4 +6,14 @@
 Raven.configure do |config|
   config.sanitize_fields = Rails.application.config.filter_parameters.map(&:to_s)
   config.release = Puzzletime.version
+
+  commit  = ENV['OPENSHIFT_BUILD_COMMIT']
+  project = ENV['OPENSHIFT_BUILD_NAMESPACE']
+
+  config.tags[:commit] = commit if commit
```

Comment:
Noch mein Senf hier: Wenn der Commit bekannt ist, würde ich den unbedingt an `config.release` anhängen (zB. `#{Puzzletime.version}_#{commit}`. So können Issues im Sentry sauber als "Resolved in next release" markiert werden und auch andere Release spezifische Features genutzt werden.

Reply:
Cool, danke für die Info.

Umgesetzt in commit [b4a95a1](https://github.com/puzzle/puzzletime/pull/66#discussion_r332362405)